httplib.h

```diff
@@ -5381,10 +5381,14 @@ write_multipart_ranges_data(Stream &strm, const Request &req, Response &res,
 
 inline bool expect_content(const Request &req) {
   if (req.method == "POST" || req.method == "PUT" || req.method == "PATCH" ||
-      req.method == "PRI" || req.method == "DELETE") {
+      req.method == "DELETE") {
```

Comment:
Nice, thanks for the fix. Btw as explained in https://github.com/yhirose/cpp-httplib/issues/2028#issuecomment-2645912843 , we should also add `OPTIONS` here (or maybe replace `HEAD` with `OPTIONS` here, I'm not sure if the browser even use HEAD or not)

Reply:
No problem. As long as `Content-Length` is set, things should work. The explicit handling of `DELETE` is specifically for handlers with content receivers. For `OPTIONS` requests, we have no such handlers.

I cannot reproduce any issues with `OPTIONS`. If that still fails, let me know what you're doing and I'll check on my end.